### PR TITLE
Use icon repo-forked instead of repo-lock for private, forked repos

### PR DESCRIPTION
**Pull request by fnkr**
Makes more sense, because private repositories still do have the yellow background.

**Before**:
screenshot from 2016-01-31 21-39-29

**After**:
screenshot from 2016-01-31 21-40-23

### CODE REVIEW

**Comment by fnkr** [comment]
Btw. GitHub does the same :D

File: templates/explore/repo_list.tmpl [hunk 1]
```diff
@@ -3,10 +3,10 @@
 		<div class="item">
 			<div class="ui header">
 				<a class="name" href="{{AppSubUrl}}/{{if .Owner}}{{.Owner.Name}}{{else if $.Org}}{{$.Org.Name}}{{else}}{{$.Owner.Name}}{{end}}/{{.Name}}">{{if $.PageIsExplore}}{{.Owner.Name}} / {{end}}{{.Name}}</a>
-				{{if .IsPrivate}}
-					<span class="text gold"><i class="icon octicon octicon-lock"></i></span>
-				{{else if .IsFork}}
+				{{if .IsFork}}
```
Comment (Unknwon) [inline templates/explore/repo_list.tmpl]: I think should still show its private first.

We can add a small line to say where it forks from, but after changed this, we can't tell it is private or not.

Comment (fnkr) [inline templates/explore/repo_list.tmpl]: @Unknwon I reverted the second changeset.

**Comment by Unknwon** [comment]
Thanks!

**Comment by Unknwon** [comment]
Updated: https://try.gogs.io/

**Pull request merged**